content/200-concepts/200-database-connectors/07-mongodb.mdx

```diff
@@ -137,19 +137,27 @@ import { ObjectId } from 'bson'
 const id = new ObjectId()
 ```
 
-## Considerations
+## Limitations and known issues
 
-There are some considerations that come with adopting Prisma into your MongoDB project, these include:
+This section covers any limitations and known issues with the MongoDB connector.
 
-- Currently no plans to add support for [Prisma Migrate](/concepts/components/prisma-migrate) as MongoDB projects don't rely on internal schemas where changes need to be managed with an extra tool
-  - Management of `@unique` indexes is realized through `db push`
-- [`@@id`](/reference/api-reference/prisma-schema-reference#id-1) and [autoincrement](/reference/api-reference/prisma-schema-reference#generate-auto-incrementing-integers-as-ids) are **not** supported.
+### No support for Prisma Migrate
+
+Currently there are no plans to add support for [Prisma Migrate](/concepts/components/prisma-migrate) as MongoDB projects do not rely on internal schemas where changes need to be managed with an extra tool. Management of `@unique` indexes is realized through `db push`.
+
+### No support for <inlinecode>@@id</inlinecode> and <inlinecode>autoincrement()</inlinecode>
+
+The [`@@id`](/reference/api-reference/prisma-schema-reference#id-1) attribute and [`autoincrement()`](/reference/api-reference/prisma-schema-reference#generate-autoincrementing-integers-as-ids) function are not supported.
 
 <!--For more information, refer to [Using Prisma with MongoDB](/content/300-guides/050-database/870-using-prisma-with-mongodb.mdx).-->
 
-## Troubleshooting
+### Cyclic references and referential actions
+
+If you have cyclic references in your models, either from self-relations or a cycle of relations between models, and you use [referential actions](/concepts/components/prisma-schema/relations/referential-actions), you must set a referential action of `NoAction` to prevent an infinite loop of actions.
+
+See [Special rules for referential actions](/concepts/components/prisma-schema/relations/referential-actions/special-rules-for-referential-actions) for more details.
 
-### Error: Transactions are not supported by this deployment
```

Comment:
People search for this error message. If this is not in the headline any more, it should be prominently featured at the beginning of the section I would suggest.

Reply:
Thanks, added the text of the message back in